frontend/src/pages/main/Main.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Main: React.FC = () => {
+    return (
+        <p>
+            Main
+        </p>
+    );
+}
+
+export default Main;
```

Comment:
structure of your pages inside main folder don't correspond with your design and routesConfig.